lib/pages/Cases.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class Cases extends StatefulWidget {
+  @override
+  _CasesState createState() => _CasesState();
+}
+
+class _CasesState extends State<Cases> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('COVID-19 Tracker: Zamboanga City'),
+        centerTitle: true,
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text(
+              'Cases',
+              style: TextStyle(
+                  color: Colors.red[300],
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20.0,),
+                  textAlign: TextAlign.center,
```

Comment:
Just set up this in `ThemeData` to reduce this kind of code in pages.

Reply:
@kusman28 set up the color of the text in `ThemeData` then remove `color` in every text unless needed to changed.